src/server/routes/api.js

```diff
@@ -63,4 +63,16 @@ router.post('/buckets', upload.array('screenshots[]', 50), errorChecking(
   },
 ));
 
+router.get('/buckets', errorChecking(
```

Comment:
I think it's not good to mix graphQL and REST, for the CLI I think it's better to be exclusively on REST for now. And I think it's better to separate endpoints.